### PR TITLE
testing/rakudo: upgrade to 2018.05

### PR DESCRIPTION
**Pull request by CurtTilmes**
Upgrade moarvm, nqp, and rakudo to version 2018.05, must upgrade all three together.

### CODE REVIEW

File: testing/nqp/APKBUILD [hunk 1]
```diff
@@ -15,11 +15,6 @@ subpackages="$pkgname-doc"
 source="${pkgname}-${pkgver}.tar.gz::https://github.com/perl6/nqp/archive/${pkgver}.tar.gz"
 builddir="$srcdir"/"$pkgname"-"$pkgver"
 
-# FIXME: some tests fail on armhf and x86, need to investigate more
-case "$CARCH" in
-	armhf | x86) options="$options !check";;
```
Comment (andypost) [inline testing/nqp/APKBUILD]: does it builds now?

Comment (CurtTilmes) [reply testing/nqp/APKBUILD]: It does on my host.  It does on Travis-CI.  Will it on the alpine build host?  I don't know.

Comment (andypost) [reply testing/nqp/APKBUILD]: I mean comment tells that on this arches some tests fails, I have only x86_64 and arm8 so can't tests

**Comment by CurtTilmes** [comment]
I have not tried it on other architectures, but I figured we should try the checks on other architectures to see what happens.

**Comment by CurtTilmes** [comment]
Superseded by version 2018.06, #4621

**Pull request closed without merging**